pgrx-examples/custom_types/custom_types.control

```diff
@@ -3,3 +3,4 @@ default_version = '@CARGO_VERSION@'
 module_pathname = '$libdir/custom_types'
 relocatable = false
 superuser = false
+trusted = false
```

Comment:
In a case where this was, e.g.
```
superuser = false
trusted = true
```
this would do nothing. While Postgres notes this is simply "irrelevant", I think pgrx should error, or at least warn on that case, rather than permitting it to pass by unremarked.

Reply:
Thanks. I added another type of error for this case.